### DISCUSSION TITLE
Replying to "Expect: 100-continue"

### DISCUSSION THREAD

**Discussion by ericdevries** [dropwizard/dropwizard#5315] [General]
Hello, my (non-dropwizard) client sends the header "Expect: 100-continue", but dropwizard does not seem to reply properly with a 100-continue header or even let me do it manually. How can I instruct dropwizard to do this?

**Comment by zUniQueX** [comment]
Hi @ericdevries. Sending the appropriate response is in the scope of the webserver, i.e. Jetty. The [documentation](https://www.eclipse.org/jetty/documentation/jetty-9/index.html#jetty-1xx-responses) from Jetty on this topic is somewhat small.

If the webserver doesn't correctly respond in the default configuration, you can manually add a [`Filter`](https://www.dropwizard.io/en/latest/manual/core.html#servlet-filters) to your application where you can decide, if your application would want to handle the request and therefore reply manually with the appropriate response.

**Comment by skradel** [comment]
Going on memory, I recall Dropwizard/Jetty/etc. correctly supporting 100 Continue in connection with cURL or httpie clients.

**Comment by joschi** [comment]
> Hello, my (non-dropwizard) client sends the header "Expect: 100-continue", but dropwizard does not seem to reply properly with a 100-continue header or even let me do it manually.

@ericdevries Could you please provide a minimal example of your Dropwizard application (and the JAX-RS resources) handling the request and a client request, ideally with cURL or as Java code?
